inference-engine/src/transformations/include/ngraph_ops/deconvolution_ie.hpp

```diff
@@ -29,6 +29,7 @@ class TRANSFORMATIONS_API DeconvolutionIE : public Op {
                     const Strides& dilations,
                     const CoordinateDiff& pads_begin,
                     const CoordinateDiff& pads_end,
+                    const element::Type output_type,
```

Comment:
we don't need it

Reply:
Don't understand

Reply:
I was wrong, was discussed offline - it's not an issue, thanks!